build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -3,6 +3,7 @@
   <Import Project="test/ProjectsToTest.props" />
   <Import Project="test/TestPackageProjects.targets" />
   <Import Project="test/TestAssetProjects.targets" />
+  <Import Project="test/TestProjects.targets" />
```

Comment:
How should we think about ProjectsToTest.props vs TestProjects.targets?  Why are these files named differently?  Should Test.targets only need to import one of them (the .targets file) and then TestProjects.targets imports the .props file?